controllers/front/OrderFollowController.php

```diff
@@ -63,7 +63,7 @@ public function postProcess()
                 Tools::redirect('index.php?controller=order-detail&id_order=' . $id_order . '&errorNotReturnable');
             }
             if ($order->id_customer != $this->context->customer->id) {
-                die(Tools::displayError());
+                Tools::redirect('index.php?controller=order-detail&id_order=' . $id_order . '&errorNotReturnable');
```

Comment:
It looks like a behavior change, not a pure message improvement.

Reply:
Yep I wrote it in the PR description. Seemed better to me than revealing what is the issue.

Reply:
I see it now. I'm not sure about this change, so we need other reviewers' idea.

Reply:
@matks Wdyt about this behavior change ?

Reply:
It makes sense. The only suggestion I'd have would be to move the condition to the first `if`.